Source/WTF/wtf/DateMath.cpp

```diff
@@ -647,6 +647,8 @@ double parseES5Date(std::span<const LChar> dateString, bool& isLocalTime)
         if (isSingleDigit && hasTSymbol)
             return std::numeric_limits<double>::quiet_NaN();
 
+        isLocalTime = hasTSymbol;
+
```

Comment:
This change does not have any effect, right? Because `parseES5TimePortion` will set `false` to `isLocalTime`.